src/_tests/utils.ts

```diff
@@ -137,3 +138,28 @@ export class Stats {
 
 export const stats = new Stats(_guard);
 */
+
+export function convertToZone(zoneString: string): Zone {
+    switch (zoneString) {
+        case '0x00':
+            return Zone.Cyprus1;
+        case '0x01':
+            return Zone.Cyprus2;
+        case '0x02':
+            return Zone.Cyprus3;
+        case '0x10':
+            return Zone.Paxos1;
+        case '0x11':
+            return Zone.Paxos2;
+        case '0x12':
+            return Zone.Paxos3;
+        case '0x20':
+            return Zone.Hydra1;
+        case '0x21':
+            return Zone.Hydra2;
+        case '0x22':
+            return Zone.Hydra3;
+        default:
+            throw new Error(`Invalid zone string: ${zoneString}`);
+    }
+}
```

Comment:
What does this function do? It appears to just return the same value that is passed in as the `zoneString`.